src/screens/Onboarding/steps/terms.js

```diff
@@ -57,73 +76,35 @@ function OnboardingStepTerms({ navigation }: *) {
           <Touchable
             event="TermsAcceptSwitch"
             onPress={onSwitch}
-            style={[styles.switchRow]}
-          >
-            <CheckBox style={styles.checkbox} isChecked={toggle} />
-            <LText semiBold style={styles.switchLabel}>
-              <Trans i18nKey="Terms.switchLabel">
-                {""}
-                <LText
-                  semiBold
-                  onPress={() => Linking.openURL(urls.terms)}
-                  color="live"
-                />
-                {""}
-              </Trans>
-            </LText>
-          </Touchable>
-          <Touchable
-            event="TermsAcceptSwitchPrivacy"
-            onPress={onSwitchPrivacy}
             style={styles.switchRow}
           >
-            <CheckBox style={styles.checkbox} isChecked={togglePrivacy} />
+            <CheckBox style={styles.checkbox} isChecked={toggle} />
             <LText semiBold style={styles.switchLabel}>
-              <Trans i18nKey="Terms.switchLabelPrivacy">
-                {""}
-                <LText
-                  semiBold
-                  onPress={() =>
-                    Linking.openURL(urls.privacyPolicy[locale || "en"])
-                  }
-                  color="live"
-                />
-                {""}
-              </Trans>
+              <Trans i18nKey="Terms.switchLabelFull" />
             </LText>
           </Touchable>
           <Button
             event="Onboarding - ToU accepted"
             type="primary"
-            disabled={!toggle || !togglePrivacy}
+            disabled={!toggle}
             onPress={next}
             title={<Trans i18nKey="Terms.cta" />}
           />
         </View>
       }
     >
       <TrackScreen category="Onboarding" name="Terms" />
-      {markdown ? (
-        <SafeMarkdown markdown={markdown} />
-      ) : error ? (
-        <View>
-          <GenericErrorView
-            error={error}
-            withIcon={false}
-            withDescription={false}
-          />
-          <ExternalLink
-            text={<Trans i18nKey="Terms.read" />}
-            onPress={() => Linking.openURL(url)}
-            event="OpenTerms"
-          />
-          <View style={styles.retryButton}>
-            <RetryButton onPress={retry} />
-          </View>
-        </View>
-      ) : (
-        <ActivityIndicator />
-      )}
+      <LinkBox
+        style={styles.linkBoxSpacing}
+        text={<Trans i18nKey="Terms.service" />}
+        url={urls.terms}
```

Comment:
you need to redirect to the urls according to the users locale.
english is the default but french, spanish, russian and chinese have their own url.
Have a look at the useTerms function to understand it a bit better